public/js/reducers/user.js

```diff
@@ -24,5 +24,13 @@ export default function user(state, action) {
     });
   }
 
+  if (action.type === actionTypes.RX_BRAINTREE_TOKEN) {
+    // As we want to stay signed in if we are
+    // this reducer inherits state.
+    return Object.assign({}, defaults, state, {
+      braintreeToken: action.user.braintreeToken,
```

Comment:
Yeah this was what I was asking about yesterday. I'm going to leave this for now as it's consistent with everything else we've been doing. It makes it very clear where things have come from. Let's discuss this and if we still want to change it we can do so in a follow-up.